src/main/java/kappzzang/jeongsan/repository/TeamRepository.java

```diff
@@ -2,17 +2,17 @@
 
 import java.util.List;
 import kappzzang.jeongsan.domain.Team;
-import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.query.Param;
 
 public interface TeamRepository extends JpaRepository<Team, Long> {
 
-    @EntityGraph(attributePaths = {"teamMemberList.member"})
-    @Query("SELECT t FROM Team t " +
-        "JOIN t.teamMemberList tm " +
-        "WHERE tm.member.id = :memberId " +
-        "AND t.isClosed = :isClosed")
+    @Query("SELECT DISTINCT t FROM Team t " +
+       "JOIN t.teamMemberList tm " +
+       "JOIN FETCH t.teamMemberList allMembers " +
```

Comment:
P2: t.teamMemberList가 두 번 join 되고 있는데 의도하신 건가요?

Reply:
JOIN FETCH의 경우 Team 도메인에 연결된 TeamMemberList를 불러오기 위함입니다. 그리고 JOIN은 memberId 필터링을 위함입니다. 다른 방법으로는 WHERE 조건에 IN을 이용해 서브쿼리를 사용할 수 있습니다. 이 쿼리로 수정할까요?

```suggestion
      @Query("SELECT DISTINCT t FROM Team t " +
       "JOIN FETCH t.teamMemberList tm " +
       "JOIN FETCH tm.member " +
       "WHERE t.id IN (SELECT tt.id FROM Team tt JOIN tt.teamMemberList ttm WHERE ttm.member.id = :memberId) " +
       "AND t.isClosed = :isClosed")
```

Reply:
JOIN이 없도록 쿼리문을 바꾼 다음 테스트해보겠습니다

Reply:
아 넵 알겠습니다
그대로 해도 될 것 같습니다

Reply:
아 넵 알겠습니다
그대로 해도 될 것 같습니다